reconfigure.bash

```diff
@@ -11,7 +11,7 @@ function usage {
 
 BUILDFARM_DEPLOYMENT_PATH=/root/buildfarm_deployment
 BUILDFARM_DEPLOYMENT_URL=https://github.com/ros-infrastructure/buildfarm_deployment.git
-BUILDFARM_DEPLOYMENT_BRANCH=master
+BUILDFARM_DEPLOYMENT_BRANCH=jenkins-lts-upgrade
```

Comment:
This branch name change should be reverted just before merging.

Reply:
I've truncated the pull request removing the commits that change this branch name.